x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/AggregationOperator.java

```diff
@@ -150,4 +163,79 @@ public String toString() {
         sb.append("aggregators=").append(aggregators).append("]");
         return sb.toString();
     }
+
+    @Override
+    public Operator.Status status() {
+        return new Status(aggregationNanos, pagesProcessed);
+    }
```

Comment:
I worried that returning a new status here is going to break older versions. I'm putting together a mixed version test check.

Reply:
Indeed - I can make this fail. I'll fix it.

Reply:
I implemented `VersionedNamedWriteable` on the status objects so they now claim the first version that they support. If we try to serialize to a remove that doesn't support that status object I just pretend the operator didn't return any status.